app/cafe.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f"{visitor["name"]} is not vaccinated.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(
+                "All friends should be vaccinated")
```

Comment:
The error message 'All friends should be vaccinated' is not descriptive enough for the `OutdatedVaccineError`. Consider including the visitor's name in the message to make it more descriptive, e.g., `f"{visitor['name']}'s vaccine is expired."`